jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/rabbitmq/RedisConfiguration.java

```diff
@@ -90,16 +85,16 @@ public JedisConnectionFactory jedisConnectionFactory(RedisProperties properties)
 	/**
 	 * Configures the cache manager
 	 * @param jedisConnectionFactory A jedisConnectionFactory
-	 * @param accessProperties Custom access properties
+	 * @param rabbitMqOutputProperties rabbitMq Output Properties
 	 * @return
 	 */
 	@Bean(name = "Document")
 	@ConditionalOnMissingBean(CacheManager.class)
-    public CacheManager cacheManager(JedisConnectionFactory jedisConnectionFactory) {
+    public CacheManager cacheManager(JedisConnectionFactory jedisConnectionFactory, RabbitMqOutputProperties rabbitMqOutputProperties) {
```

Comment:
please run the sample app to make sure it works

Reply:
ran the sample. confirm it works.